### ISSUE TITLE
feature request: read data from an I Q stream for openwebrx

### ISSUE THREAD

**Issue by hubertmeier** [merbanan/rtl_433#2110]
Hi,

it would be cool to see your work integrated in openwebrx, a very nice web-based receiver which supports many input devices and could display decoded data in the Webinterface. Openwebrx delivers I/Q-Streams to the decoder and gets back the output of the decoder to display it in the Webinterface.

rtl_433 seems not to accept I/Q-Streams, therefore your project can not be used in openwebrx...

you will find more informations:

https://github.com/jketterl/openwebrx

Thank you

Hubert

**Comment by merbanan** [comment]
Well we do support rtl_tcp input. I concider that beeing streamad I/Q.

**Comment by zuckschwerdt** [comment]
Good point about `rtl_tcp`. Also we have streaming input from stdin with e.g. `CU8:-` as filename (https://triq.org/rtl_433/OPERATION.html#loaders-and-dumpers) and the JSON output should be suitable.

Some `Popen` module like https://github.com/jketterl/openwebrx/blob/develop/csdr/module/drm.py should be workable.
Or even better use some networked I/O, e.g. rtl_tcp in and UDP out.

I don't have time to implement something, but let us know if anything should be improved to make integration easier.

**Comment by gdt** [comment]
It sounds like this can be done with rtl_tcp or stdin, and this issue does not have a pointer to a specific protocol spec and/or library that is not implemented.  I think everybody at rtl_433 would like to see  something happen, but we need a concrete spec, and preferably that should be some semi-widely-implemented lingua franca.  Please update to be precise and actionable.
